catalog/task/validate-single-component/0.1/validate-single-component.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: validate-single-component
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task validates that the snapshot only contains a 
+    single component. A pipeline should fail otherwise.
```

Comment:
This is a task, not a pipeline. Also, "should" is not the right verb. So it should probably say: "The task will fail otherwise."